spec/features/invites_feature_spec.rb

```diff
@@ -44,6 +47,10 @@ def expect_name_and_email_fields_to_be_filled(name, email)
   def submit_register_form(password)
     fill_in "Password", with: password
     fill_in "Confirm Password", with: password
+    fill_in "Credit Card Number", with: '4242424242424242'
+    fill_in "Security Code", with: '123'
+    select "7 - July", from: 'exp-month'
+    select "2018", from: 'exp-year'
     click_button "Sign Up"
```

Comment:
After you submit registration form, you will have to wait for Ajax response from Stripe (to receive token) - I would recommend putting `sleep 3` here instead of increasing default sleep in `spec_helper.rb`.